app/src/main/java/com/devshish/internship/presentation/service/player/MediaSessionCallback.kt

```diff
@@ -49,6 +112,21 @@ class MediaSessionCallback(
         )
         exoPlayer.play()
         Timber.d("onPlay")
+
+        notificationCompatBuilder.clearActions()
+        notificationCreateEvent(
+            notificationCompatBuilder
+                .addAction(
+                    NotificationCompat.Action(
+                        R.drawable.exo_icon_pause,
+                        null,
+                        MediaButtonReceiver.buildMediaButtonPendingIntent(
+                            context,
+                            PlaybackStateCompat.ACTION_PLAY_PAUSE
+                        )
+                    )
+                )
```

Comment:
I thing you can move this into some method, because there are a lot of duplicating code, and only icon is what changes